src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseDataValidatorController.java

```diff
@@ -38,7 +37,6 @@ public CaseDataValidatorController(
         this.midEventCallback = midEventCallback;
     }
 
-    @Transactional
     @PostMapping(
```

Comment:
do we need PostMapping here?

Reply:
well is it doing a invokeMidEventCallback() operations ,  it is not in the scope of  the ticket  removing it . we can do a call evaluated together :) and maybe we can create a new ticket.